client/js/components/App.react.jsx

```diff
@@ -16,24 +18,51 @@ export default class App extends React.Component {
     constructor(props) {
         super(props);
         this.state = {park: null,
-                      parkList: []};
-        getParksJson().then((data) => this.setState({parkList: data}));
+                      parkList: [],
+                      parkByID: {}};
+        getParksJson().then((data) => {
+            var parkByID = {}
+            data.forEach((current, index) => {
+                // NOTE: Coordinates are lng lat here
```

Comment:
Btw the reason they are lng lat is because they are actually from GeoJson coordinates, which "follow x, y, z order (easting, northing, altitude)"